item-spec/common-metadata.md

```diff
@@ -207,3 +208,23 @@ optical and short-wave IR bands are all 30 meters, but the panchromatic band is
 PlanetScope Ortho Tile Product has an `gsd` of 3.7 (or 4 if rounding), even though the pixel size of the images is 3.125.
 For example, one might choose for WorldView-2 the Multispectral 20° off-nadir value of 2.07
 and for WorldView-3 the Multispectral 20° off-nadir value of 1.38.
+
+### Link Object
+
+This object describes a relationship with another entity. Data providers are advised to be liberal
+with the links section, to describe things like the Catalog an Item is in, related Items, parent or
+child Items (modeled in different ways, like an 'acquisition' or derived data).
+It is allowed to add additional fields such as a `title` and `type`.
+
+| Field Name | Type   | Description                                                                                                                                                                    |
+| ---------- | ------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| href       | string | **REQUIRED.** The actual link in the format of an URL. Relative and absolute links are both allowed. [Trailing slashes are significant.](../best-practices.md#consistent-uris) |
+| rel        | string | **REQUIRED.** Relationship between the current document and the linked document. See chapter "Relation types" for more information.                                            |
+| type       | string | [Media type](../catalog-spec/catalog-spec.md#media-types) of the referenced entity.                                                                                            |
+| title      | string | A human readable title to be used in rendered displays of the link.                                                                                                            |
+| method     | string | The HTTP method required to be supported by the target resource. `GET` by default                                                                                              |
+| headers    | object | The HTTP headers required to be sent by the target resource.                                                                                                                   |
```

Comment:
We don't use the object type in the spec, instead we use `Map<string, *>` (applies to headers and body for now)

Reply:
`Map<string, string|array<string>>`

`{a: "b"}`
translates to:
`a: b`

`{a: ["b", "c"]}`
translates to:
```
a: b
a: c
```

This could hopefully also be clarified in the STAC API spec.